tests.js

```diff
@@ -208,4 +208,16 @@ QUnit.test('String concatenation on array values without comma', function(
   assert.equal(obj.a[0], '1 2 3 4');
 });
 
+QUnit.test('String concatenation in field keys', function(assert) {
+  var obj = parseHocon(`{
+    a: 42,
+    a b : 3
+  }`);
```

Comment:
oh, bummer.. ok, might add the build step in the future, for now i'll fix the code.